docs/sources/flow/getting-started/migrating-from-prometheus.md

```diff
@@ -0,0 +1,187 @@
+---
```

Comment:
We should replace instances of "Grafana Agent Flow" throughout this file with "Grafana Agent flow mode" or just "flow mode"